### PR TITLE
Update Arwen with hotfix

### PR DESCRIPTION
**Pull request by camilbancioiu**
This PR updates dependency on Arwen v1.3.25.

### CODE REVIEW

**Review by sasurobert — COMMENTED** [review]
go mod tidy - need to see changes in go.sum as well.

**Review by sasurobert — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -4,7 +4,7 @@ go 1.13
 
 require (
 	github.com/ElrondNetwork/arwen-wasm-vm/v1_2 v1.2.26
-	github.com/ElrondNetwork/arwen-wasm-vm/v1_3 v1.3.24
+	github.com/ElrondNetwork/arwen-wasm-vm/v1_3 v1.3.25
```
Comment (iulianpascalau) [inline go.mod:7]: does the fix in arwen is backwards compatible?

Comment (iulianpascalau) [reply go.mod:7]: According to the latest commit, it looks like it should not break the backwards compatibility.

**Review by iulianpascalau — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]
System test passed.

**Pull request merged**